docs/download_files.md

```diff
@@ -293,7 +294,7 @@ In addition, the `demux_cell_count_estimate` column will contain an estimate of
 
 ## Spatial transcriptomics libraries
 
-If a sample includes a library processed using spatial transcriptomics, the spatial transcriptomics output files will be available as a separate download from the single-cell/single-nuclei gene expression data.
+If a sample includes a library processed using spatial transcriptomics, the spatial transcriptomics output files will be available as a separate download from the single-cell/single-nuclei gene expression data by selecting "Spatial" as the modality (see more on {ref}`modalities <download_options:modalities>`). 
```

Comment:
Noting that I did not rephrase the part "...as a separate download..." but I do think some more rephrasing would be helpful, since what do we mean by download? If it's part of the same custom dataset, it would be the same download just in a different file. I think the rephrase to implement here is just changing "download" -> "file" but if you have a different idea go for it!

```suggestion
If a sample includes a library processed using spatial transcriptomics, you can obtain the spatial transcriptomics output files as a separate download from the single-cell/single-nuclei gene expression data by selecting "Spatial" as the modality (see more on {ref}`modality download options<download_options:modalities>`). 
```